### DISCUSSION TITLE
Markup Schema 形式中，非标单组件的的联动怎么控制

### DISCUSSION THREAD

**Discussion by WuHuSa** [alibaba/formily#2676] [Q&A 2.x]
比如，字段 a 控制字段 b 的显示/隐藏，同时有个Button组件要跟随 b 一起显示/隐藏，x-reactions 可以处理字段的联动关系，此处的 Button 这种非字段，要怎么控制其显示/隐藏

**Comment by WuHuSa** [comment]
除了自定义组件，有别的方式吗

**Reply by janryWang** [reply]
void字段也能使用x-reactions的
